### PR TITLE
Upload files to deposition and create record in one go.

### PR DESCRIPTION
**Pull request by arshul**

### CODE REVIEW

**Comment by arshul** [comment]
Hey @ivotron please review this too.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,58 @@
...
+            {
+                "name": "Arshul, Arshul"
+            }
+        ]
+    }
+}
+```
+### Usage
+Keep the metadata in a `json` file and set the path to this `json` file as `ZENODO_METADATA_PATH` environment variable.
+Put the files to be uploaded in a directory and set the path to this directory as `UPLOAD_FILES_DIRECTORY_PATH` environment variable.
```
Comment (ivotron) [inline README.md]: can we rename this to `ZENODO_UPLOAD_FOLDER` or `ZENODO_UPLOAD_PATH` . Looks like there's a convention of prefixing env variable names with the name of service/app being used.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,58 @@
 # Github Actions for Zenodo
+Documentation for each action could be found in their respective directories.
 
-Documentation for each action could be found in their respective directories.
\ No newline at end of file
+## Create a Record
```
Comment (ivotron) [inline README.md:4]: Is this README supposed to be on the root of the folder?

**Review by ivotron — CHANGES_REQUESTED** [review]
thanks @arshul! please take a look at the reviews. One more general question I have is: would it make sense to combine the `create` and `upload` actions, so that if an upload action is executed for the first time, a deposition is created? Or put it differently, what would the value of having a separate `create` action be?

File: README.md (hunk 2 above)
Comment (arshul) [reply README.md:4]: Create Record is the combination of all the actions, and there is no dedicated directory for that, so to give details about the workflow for publishing a record I have to do that in the root readme. You can tell me if it needs to be somewhere else.

File: README.md (hunk 1 above)
Comment (arshul) [reply README.md]: I have also read somewhere to append _PATH at the end of a path variable, so `ZENODO_UPLOAD_PATH` would be good. I will change it to that.

**Comment by arshul** [comment]
>would it make sense to combine the create and upload actions, so that if an upload action is executed for the first time, a deposition is created? 

`create` and `upload` can always be combined in the workflow while executing, it is more modular and flexible this way IMO. 

>what would the value of having a separate create action be?

yeah, that kinda makes sense, I could do something like, change the `upload.workflow` to create a deposition first and then upload to that deposition.
Or if you really want to combine the two, i.e make them in a single action, I can try to implement that also.

File: README.md (hunk 1 above)
Comment (ivotron) [reply README.md]: sounds good

File: README.md (hunk 2 above)
Comment (ivotron) [reply README.md:4]: I see. So this is showing a complete example. Yeah, this makes sense.

**Comment by ivotron** [comment]
> `create` and `upload` can always be combined in the workflow while executing, it is more modular and flexible this way IMO.

I agree.

**Review by ivotron — APPROVED** [review]

**Comment by ivotron** [comment]
@arshul the only change missing is that one of naming the environment variable `ZENODO_UPLOAD_PATH`. After that, this should be ready to be merged

**Comment by arshul** [comment]
Changed the path.

**Comment by ivotron** [comment]
thanks a lot!

**Pull request merged**